addons/account/migrations/13.0.1.1/post-migration.py

```diff
@@ -0,0 +1,719 @@
+        invoice_source_email, invoice_partner_display_name,
+        invoice_cash_rounding_id, create_uid, create_date, write_uid,
+        write_date) = (
+        ai.message_main_attachment_id, ai.access_token,
+        COALESCE(ai.number, ai.move_name, am.name), COALESCE(ai.date, am.date),
+        ai.reference, ai.comment, ai.type, ai.journal_id, ai.company_id,
+        ai.currency_id, ai.partner_id, ai.commercial_partner_id,
+        ai.amount_untaxed, ai.amount_tax, ai.amount_total, ai.residual,
+        ai.amount_untaxed_signed, COALESCE(ai.amount_total_signed,
+        ai.amount_total_company_signed), ai.residual_signed,
+        ai.fiscal_position_id, ai.user_id, CASE WHEN ai.state = 'cancel'
+        THEN 'cancel' ELSE 'posted' END, CASE WHEN ai.state IN (
+        'in_payment', 'paid') THEN ai.state ELSE 'not_paid' END,
+        ai.date_invoice, ai.date_due, ai.name, ai.sent, ai.origin,
+        ai.payment_term_id, ai.partner_bank_id, ai.incoterm_id,
+        ai.source_email, ai.vendor_display_name, ai.cash_rounding_id,
+        ai.create_uid, ai.create_date, ai.write_uid, ai.write_date)
+        FROM account_invoice ai
+        WHERE am.old_invoice_id = ai.id AND ai.state not in ('draft', 'cancel')
+        """,
+    )
+    openupgrade.logged_query(
+        env.cr, """
+        INSERT INTO account_move (message_main_attachment_id, access_token,
+        name, date, ref, narration, type, journal_id, company_id, currency_id,
+        partner_id, commercial_partner_id, amount_untaxed, amount_tax,
+        amount_total, amount_residual, amount_untaxed_signed,
+        amount_total_signed, amount_residual_signed, fiscal_position_id,
+        invoice_user_id, state, invoice_payment_state, invoice_date,
+        invoice_date_due, invoice_payment_ref, invoice_sent, invoice_origin,
+        invoice_payment_term_id, invoice_partner_bank_id, invoice_incoterm_id,
+        invoice_source_email, invoice_partner_display_name,
+        invoice_cash_rounding_id, old_invoice_id, create_uid, create_date,
+        write_uid, write_date)
+        SELECT message_main_attachment_id, access_token,
+        COALESCE(number, move_name, name, '/'), COALESCE(date, date_invoice,
+        write_date), reference, comment, type, journal_id, company_id,
+        currency_id, partner_id, commercial_partner_id, amount_untaxed,
+        amount_tax, amount_total, residual, amount_untaxed_signed,
+        amount_total_signed, residual_signed, fiscal_position_id, user_id,
+        state, 'not_paid', date_invoice, date_due, name, sent, origin,
+        payment_term_id, partner_bank_id, incoterm_id, source_email,
+        vendor_display_name, cash_rounding_id, id, create_uid, create_date,
+        write_uid, write_date
+        FROM account_invoice ai
+        WHERE ai.state in ('draft', 'cancel')
+        RETURNING old_invoice_id, id""",
+    )
+    ids = [(x[0], x[1]) for x in env.cr.fetchall()]
```

Comment:
```suggestion
    ids = env.cr.fetchall()
```
No point converting a list of tuple to a list of tuples.